README.md

```diff
@@ -31,37 +31,40 @@ Colleagues may collaborate on a spreadsheet that looks like this:
 
 You can export your spreadsheet to CSV, run it through the ECS mapper,
-and get your starter pipelines generated.
+and generate your starter pipelines.
 
-Note that this tool generates starter pipelines. They only do the rename/copy
-operations and some field format adjustments. It's up to you to integrate them
+Note that this tool generates starter pipelines. They only do field rename and copy
+operations as well as some field format adjustments. It's up to you to integrate them
 in a complete pipeline that ingests and outputs the data however you need.
 
-Scroll down to the [Examples](#examples) section below, to get right to a
+Scroll down to the [Examples](#examples) section below to get right to a
 concrete example you can play with.
 
 ## Maturity
 
 This code is a proof of concept and is not officially supported.
-The pipelines generated by this tool are not meant to be complete, nor production-ready.
+The pipelines generated by this tool are likely not complete and probably need more
+testing and validation before they are ready for production.
 They are simply meant to give you a head start in mapping various sources to ECS.
 
 ## CSV Format
 
-Here's more details on the CSV format supported by this tool. Since mapping
-spreadsheets are primarily used by humans, it's totally fine to have as many columns
+Here are more details on the CSV format supported by this tool. Since mapping
+spreadsheets are used by humans, it's totally fine to have as many columns
 as you need in your spreadsheets/CSV. Only the following columns will be considered:
 
 | column name | required | allowed values | notes |
 |-------------|----------|----------------|-------|
 | source\_field | required |  | A dotted Elasticsearch field name. Dots represent JSON nesting. Lines with empty "source\_field" are skipped. |
 | destination\_field | required |  | A dotted Elasticsearch field name. Dots represent JSON nesting. Can be left empty if there's no copy action (just a type conversion). |
-| format\_action | optional | to\_float, to\_integer, to\_string, to\_boolean, to\_array, uppercase, lowercase, (empty) | Simple conversion to apply to the field value. |
+| format\_action | optional | to\_float, to\_integer, to\_string, to\_boolean, to\_array, to\_timestamp\_unix, to\_timestamp\_unix\_ms, uppercase, lowercase, (empty) | Simple conversion to apply to the field value. |
 | copy\_action | optional | rename, copy, (empty) | What to do with the field. If left empty, default action is based on the `--copy-action` flag. |
 
 You can start from this
 [spreadsheet template](https://docs.google.com/spreadsheets/d/1m5JiOTeZtUueW3VOVqS8bFYqNGEEyp0jAsgO12NFkNM). Make a copy of it in your Google Docs account, or download it as an Excel file.
 
+When the destination field is @timestamp, then we always enforce an explicit date ```format\_action``` to avoid conversion problems downstream. If no format is provided, then UNIX_MS is used.
```

Comment:
I'm wondering if we should also note this in the help output? Maybe it's fine to just have it here?